src/vivarium_public_health/population/mortality.py

```diff
@@ -97,19 +126,30 @@ def setup(self, builder: Builder) -> None:
         self.clock = builder.time.clock()
 
         self.cause_specific_mortality_rate = self.get_cause_specific_mortality_rate(builder)
-        self.mortality_rate = self.get_mortality_rate(builder)
-
-        self.all_cause_mortality_rate = self.get_all_cause_mortality_rate(builder)
-        self.life_expectancy = self.get_life_expectancy(builder)
 
-        self._raw_unmodeled_csmr = self.get_raw_unmodeled_csmr(builder)
         self.unmodeled_csmr = self.get_unmodeled_csmr(builder)
         self.unmodeled_csmr_paf = self.get_unmodeled_csmr_paf(builder)
+        self.mortality_rate = self.get_mortality_rate(builder)
 
     #################
     # Setup methods #
     #################
 
+    def create_lookup_tables(self, builder: "Builder") -> None:
+        """
+        Create lookup tables for the mortality component.
+
+        Parameters
+        ----------
+        builder
+            Interface to access simulation managers.
+        """
+        super().create_lookup_tables(builder)
+        self.unmodeled_cause_specific_mortality_rate = self.get_raw_unmodeled_csmr(builder)
+        self.lookup_tables[
+            "unmodeled_cause_specific_mortality_rate"
+        ] = self.unmodeled_cause_specific_mortality_rate
```

Comment:
nit: is there a reason not to just do `self.lookup_tables["unmodeled_cause_specific_mortality_rate"] = self.get_raw_unmodeled_csmr(builder)` ?

Reply:
Good call.